docker/local-build.sh

```diff
@@ -11,9 +11,12 @@ BUILD_CACHE_DIR=$WORKSPACE/bb-cache
 BUILDOS="opensuse-42.1"
 GIT_PROXY_COMMAND=oe-git-proxy
 TARGET_MACHINE="intel-corei7-64"
+GIT_COMMITTER_EMAIL=${GIT_COMMITTER_EMAIL:-$(git config --get user.email)}
+GIT_COMMITTER_NAME=${GIT_COMMITTER_NAME:-$(git config --get user.name)}
+GIT_COMMITTER_NAME=${GIT_COMMITTER_NAME:-${USER}}
```

Comment:
Don't do that. Git apply error is real error. Fix it, instead of trying to hide.